content/en/blog/sign-verify-plugins-rules/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "How we Sign and Verify Plugins and Rules"
+linktitle: "How we Sign and Verify Plugins and Rules"
+date: 2023-10-12
+author: Massimiliano Giovagnoli, Luca Guerra
+slug: sign-verify-plugins-rules
+images:
+  - /blog/how-we-sign-and-verify-plugins-and-rules/images/falcoctl-signature-featured.png
+---
+
+## Falco v0.36.0 and the SSC security
+
+The latest stable v0.36.0, alongside falcoctl 0.6.1 and the 0.7.0 Helm chart brought new features and new improvements in the supply chain security of Falco artifacts. The two main artifacts are plugins and rule sets as you already know. They're shipped in the OCI specification format and distributed through the official Falcosecurity OCI repositories.
+
+Software supply chain attacks aim at injecting malicious code into software components like the above, to compromise downstream users. These types of attacks are among the primary threats in today's threat landscape.
+
+In particular, attackers abuse trust relationships existing between the different open-source stakeholders. The increase in attacks on open-source software throughout the last few years demonstrates that attackers consider them a viable means for spreading malware.
+
+## SSC safeguards
```

Comment:
I think you need a quick introduction for this section.  

for example

the task of securing your software supply chain may seem daunting at first glance. But there are a number of ways you can help ease this effort.

Reply:
Thank you @mikegcoleman!

Reply:
```suggestion
## SSC safeguards

Securing the software supply chain may seem daunting at first glance, but there are a lot of safeguards that can be put in action. And there are ways to categorize them, and ways to prioritize them.
```